### DISCUSSION TITLE
I launched LazyVim for the first time, and this is how it greets me

### DISCUSSION THREAD

**Discussion by PgLoLo** [LazyVim/LazyVim#1545] [General]
I have been a long-time Vim user, using plain Vim for editing small files. However, for everyday Python development, I have always used full-fledged IDEs. Now, I need to learn a few new programming languages, so I decided it's the perfect opportunity to try switching to Vim, and LazyVim seems like the perfect starting point.

I installed a clean build of LazyVim and opened my small Python project with around 50 source files. I entered my first command: Space - f - F and typed the file name: run.py. The entire LazyVim hangs for at least five seconds, taking all terminal instances with it. Almost any change in the entered string repeats the entire freeze.

Perhaps, space-f-F is not the best way to open a file in the local directory. However, it all looks incredibly sad. This was literally my first entered command. Honestly, the prospects of LazyVim don't seem very promising after such an encounter.

I don't fully understand how to react to this. This is a bisic functionality of any program in 2k23: to not hang the entire application at any reason. Yet alone by performing the most natural command for text editor. Honestly, the impression that was created by all the resources I looked at and saw about the Neovim ecosystem is completely shattered. Is it even worth spending my time? Is there a chance that something will change this first impression?

**Comment by folke** [comment]
That is obviously not normal. Never heard anyone having that problem ever.
I'm not here to convince you to use LazyVim. Maybe LayVim is just not for you.

Edit: did you install all the requirements as mentioned in the installation instructions?

Especially `fd`? If `fd` is not on your system, then Telescope.nvim (the fie finder) would use regular `find` instead, which could be slow since that would not honor any `.gitignore` files.

After installing LazyVim, did you run `checkhealth` as is instructed and fix any possible issues?

**Comment by dpetka2001** [comment]
Don't expect anyone to change your impression. You're the only one that can change that, depending on your experiences and the time you put into something. If it's worth investing your time is also entirely up to you. You should first and foremost put your priorities and responsibilities ahead of everything else. I don't like answering definitively questions like these, so what I end up usually suggesting, is for someone to see the transition to Neovim as a side project in their free time. As the side project progresses, you will be able to answer for yourself if it's worth your time in the end. If you don't have any free time for side projects, that's ok too, because no one will be able to be a better judge than you. So, the question comes down to "Are you able to find some free time for a new side project or not?" If you find along the way that it's not worth it after all, then you just ditch it.
